ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/PeerDiscoveryControllerTest.java

```diff
@@ -117,11 +117,11 @@ public void bootstrapPeersRetriesSent() {
 
     controller.start();
 
-    int timeouts = 4;
+    final int timeouts = 4;
     for (int i = 0; i < timeouts; i++) {
       timer.runTimerHandlers();
     }
-    int expectedTimerEvents = (timeouts + 1) * peerCount;
+    final int expectedTimerEvents = 1 + (timeouts + 1) * peerCount;
```

Comment:
Why add the 1 here?  Is it more accurate to change `times` to `atLeast` on line 125: `verify(timer, atLeast(expectedTimerEvents)).setTimer(anyLong(), any());`?